Multisig/Data/Services/Safe Client Gateway Service/ConfirmTransactionRequest.swift

```diff
@@ -11,8 +11,10 @@ import Foundation
 struct ConfirmTransactionRequest: JSONRequest {
     var safeTxHash: String
     var signedSafeTxHash: String
+    let network: Int
+    
     var httpMethod: String { "POST" }
-    var urlPath: String { "/v1/transactions/\(safeTxHash)/confirmations" }
+    var urlPath: String { "/\(network)/v1/transactions/\(safeTxHash)/confirmations" }
```

Comment:
The naming seems inconsistent with the other occurrences where it is called `chainId` ;)

Reply:
I would rename in other places into `network` as we use this name in our model

Reply:
Yes, by mistake :)